packages/itwinui-react/src/core/Table/Table.test.tsx

```diff
@@ -4695,3 +4695,46 @@ it('should ignore top-level Header if one is passed', async () => {
     expect(cells[1].textContent).toEqual(description);
   });
 });
+
+it('should pass custom props to different parts of Table', () => {
+  const { container } = renderComponent({
+    data: [],
+    headerWrapperProps: {
+      className: 'custom-header-wrapper-class',
+      style: { fontSize: 12 },
+    },
+    headerProps: { className: 'custom-header-class', style: { fontSize: 12 } },
```

Comment:
nit: I would test different value for each different class (i.e 12, 13, 14...) just in case a style get inherited and not actually the passed in prop value and we wouldn't be able to tell when the values are the same

Reply:
Good call, I'll implement that in the test

Reply:
Added!